jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/categoryattr/CategoryAttributesDetailView.java

```diff
@@ -281,14 +283,50 @@ protected void onInit(InitEvent event) {
         initCalculatedValuesAndOptionsForm();
         initViewGrid();
     }
+    private void initDefaultEnumField() {
+        setDefaultStringValue();
+        defaultEnumField.addValueChangeListener(e -> getEditedEntity().setDefaultString(e.getValue()));
+        if(StringUtils.isNotBlank(getEditedEntity().getDefaultString())) {
+            defaultEnumField.setValue(getEditedEntity().getDefaultString());
+        }
+
+        if(!StringUtils.isBlank(categoryAttributeDc.getItem().getDefaultString())) {
+            defaultEnumField.getDataProvider().refreshAll();
+            defaultEnumField.setValue(categoryAttributeDc.getItem().getDefaultString());
+        }
+    }
+
+    private void setDefaultStringValue() {
```

Comment:
Why a method that is called `setDefaultStringValue` initializes comboBox options? It looks like an inappropriate name for the method.